crates/ruma-client-api/CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Improvements:
 - Add the `get_login_token` field to `Capabilities`, according to a
   clarification in the spec.
 - Add support for account locking, according to MSC3939.
+- Allow constructing `error::ErrorBody` outside of this crate.
```

Comment:
This would be a breaking change.

Reply:
Why would it be a breaking change? Also,

```suggestion
- Allow constructing `error::ErrorBody::NotJson` outside of this crate.
```

Reply:
Won't the compiler complain about `match ErrorBody::NotJson { bytes, deserialization_error, .. }` if there are no other fields?

Reply:
Nevermind, I just tested and it doesn't say anything (not even a clippy warning), so it's not a breaking change.

Reply:
I would expect a warning (maybe only in match, not if let?) at least by clippy, but new warnings aren't breaking changes.